src/accountsdb/db.zig

```diff
@@ -2530,10 +2614,12 @@ pub const AccountsDB = struct {
         deprecated_stored_meta_write_version: u64 = 0,
     };
 
+    pub const GenerateIncSnapshotResult = BankIncrementalSnapshotPersistence;
```

Comment:
this is stange type alias - if something changed with BankIncrementalSnapshotPersistence it would break this ...